MIGRATION.md

```diff
@@ -11,8 +15,8 @@
     - [The `label` prop only accepts a string](#the-label-prop-only-accepts-a-string)
     - [Improved `validationHint` for the `Checkbox` component](#improved-validationhint-for-the-checkbox-component)
     - [Minor fixes](#minor-fixes)
-  - [Other changes](#other-changes)
-- [From v4 to v5](#from-v4-to-v5)
+  - [Other changes](#other-changes-1)
+- [From v4.x to v5](#from-v4x-to-v5)
```

Comment:
Aligned this (half the titles were "From v*.x to v*" and the others were "From v* to v*".